src/services/mol/types.d.ts

```diff
@@ -0,0 +1,37 @@
+interface Info {
+  content: string;
+  lang: string;
+  source: string;
+}
+
+interface Family {
+  lang: string;
+  name: string;
+}
+
+export interface SpeciesItemProps {
```

Comment:
Not sure this "Props" suffix is the best way to name interfaces.
In any case, it would have to be consistent with the rest of the project.